### ISSUE TITLE
MQTT error message persists until tablet restart

### ISSUE THREAD

**Issue by exxamalte** [thanksmister/wallpanel-android#20]
I have configured Wallpanel to send regular updates via MQTT. Occasionally (about every couple of days or so), the app apparently loses its connection to the MQTT broker, and shows a persistent error message on the screen: "Couldn't subscribe to the MQTT topics, check the MQTT broker settings or your connection."
This message stays on top of the screen. Even when I leave the app, it remains on the screen.
When I "Force Stop" the app, and restart Wallpanel, the error message reappears after a few seconds.
Only when I restart the whole tablet, the message disappears, and Wallpanel successfully reconnects to my broker.
At the time of the error message, the tablet itself is still connected to my Wifi, and my MQTT broker is up and running. I've configured the MQTT broker using its IP address.

Tablet: Fire HD 8
Version: 0.8.4 Build 1 (but I have this problem for a few months already)

**Comment by thanksmister** [comment]
Yeah I’m aware of the issue and will issue a fix next week.  Thanks for reporting it.
